README.md

```diff
@@ -106,7 +106,6 @@ LoomProvider combined with Web3 supports the following `JSON RPC` calls
 - [x] eth_accounts
 - [x] eth_blockNumber
 - [x] eth_call
-- [x] eth_estimateGas
```

Comment:
Let's move this down to `Should`, and add a note that it currently always returns zero.

Reply:
@enlight Sure thing.